rust/operator-binary/src/controller.rs

```diff
@@ -857,6 +862,25 @@ async fn build_node_rolegroup_statefulset(
         .resources(merged_config.resources.clone().into());
 
     let mut pod_builder = PodBuilder::new();
+
+    // Add user configured extra volumes if any are specified
+    if let Some(extra_volumes) = &nifi.spec.extra_volumes {
+        tracing::info!(
+            "Found user specified extra volumes: {:?} these will be mounted in '{}'",
+            extra_volumes,
+            USERDATA_MOUNTPOINT
+        );
```

Comment:
I considered extracting the most relevant fields, yes. But then couldn't make up my mind on what is "most important" because depending on what the user mounts here everything might end up being important..

Reply:
it now just logs name and type